src/main/java/uk/gov/beis/els/categories/spaceheaters/model/BoilerPackagesCalculatorForm.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.beis.els.categories.spaceheaters.model;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotNull;
+import org.hibernate.validator.group.GroupSequenceProvider;
+import uk.gov.beis.els.model.meta.FieldPrompt;
+
+@GroupSequenceProvider(BoilerPackagesCalculatorFormSequenceProvider.class)
+public class BoilerPackagesCalculatorForm extends SpaceHeaterPackagesCalculatorForm{
+
+  private final PreferentialHeaterTypes preferentialHeaterType = PreferentialHeaterTypes.BOILER;
+
+  @FieldPrompt("Supplementary heat pump installed?")
+  @NotNull(message = "Specify if the package includes a supplementary heat pump")
+  private boolean hasSupplementaryHeatPump;
+
+  @FieldPrompt("Rated heat output (Prated) of the supplementary heat pump (in kW)")
+  @Digits(integer = 4, fraction = 0, message = "Enter the heat output of the supplementary heat pump",
+            groups = SupplementaryHeatPumpGroup.class)
+  private String supplementaryHeatPumpHeatOutput;
+
+  @FieldPrompt("Seasonal space heating energy efficiency of supplementary heat pump (in %)")
+  @Digits(integer = 3, fraction = 0, message = "Enter the seasonal space heating energy efficiency of supplementary heat pump",
+            groups = SupplementaryHeatPumpGroup.class)
+  private String supplementaryHeatPumpSeasonalSpaceHeatingEfficiencyPercentage;
+
+  @Override
+  public PreferentialHeaterTypes getPreferentialHeaterType() {
+    return preferentialHeaterType;
+  }
+
+  @Override
+  public String getPreferentialHeatPumpSeasonalSpaceHeatingEfficiencyColderPercentage() {
+    return null;
+  }
+
+  @Override
+  public void setPreferentialHeatPumpSeasonalSpaceHeatingEfficiencyColderPercentage(
+      String preferentialHeatPumpSeasonalSpaceHeatingEfficiencyColderPercentage) {
+  }
+
+  @Override
+  public String getPreferentialHeatPumpSeasonalSpaceHeatingEfficiencyWarmerPercentage() {
+    return null;
+  }
+
+  @Override
+  public void setPreferentialHeatPumpSeasonalSpaceHeatingEfficiencyWarmerPercentage(
+      String preferentialHeatPumpSeasonalSpaceHeatingEfficiencyWarmerPercentage) {
+  }
+
+  @Override
+  public boolean isLowTemperatureHeatPump() {
+    return false;
```

Comment:
This should return `isLowTemperatureHeatPump` right? Same with the other getters that return false.
I think this is pre-setting the radios on the screen to 'No' when they should be un-set.

Reply:
removed unused getters
But it was pre-setting the radios to no because boolean primitives got initialized to false, instead of Booleans that get null.

done